### ISSUE TITLE
Clean up the Comment Section

### ISSUE THREAD

**Issue by KadotaTerry** [barrycumbie/jobie-19#35]
@ someone when you are replying to them so that the comments appear more organized and do not seem so incoherent and scattered.

**Comment by barrycumbie** [comment]
whoa @lmowens emoji reaction. Nice. 

There are several issues about cleaning up that gist, e.g. #39, #29 and prob others. This comment one is distinct from the others so I am not suggesting to combine groups (will get too big and not get anything done). Let's keep this team (whomever it shall be) small and nimble. 

What I think is very first thing we need is a PROCESS DIAGRAM. Draw it on paper and upload the pic, use Visio or Draw.io or Lucidcharts, or whatever! 

Recall from the gist a process is what? Input-Process-Output (with set up prior and tear down after). This whole "class notes/info thing" requires a name (to define what we are actually doing) & prob. several sub-processes. Good processes have feedback loops and are half- or full-duplex (two-way vs. one-way). Super-duper processes have it where the OUTPUT of Sub-Process1 = the INPUT of Sub-Process2. Guess what happens then? We don't have to manually inspect it, copy/paste it, re-key the data...one tech can drop it off and another one can pick it up! Boom, integrated processes! 💥

Here's my super quick take on this process: 

INPUTS: Anyone from class (including Cumbie and Moon) can POST information. (Cumbie has some exclusive content, i.e., Zoom links and class topics stuff so maybe that is a separate process).

PROCESS: Information is recorded, stored, organized. 

OUTPUTS: anyone from class (this one and future classes) can readily find and use that information within context (i.e., they know who did it, why they did it, what term/date, etc.)

Current process: 
BC opens/edits Gist, randomly sprinkling in MD. Students comment on Gist. 
Info saved on Gist
Future PPL open Gist, search all over to find what they are looking for to mixed success. Hard to follow top down and info is splintered across days/topics/comments. 

What we need? 
1. A home page with links to topics, topic categories (general skills, dev resources, product documentation specific to our class/repo, fact book with sub topics of lectures and stuff like the Agile Manifesto. 
1. A process, rules about posting new info, edits, etc.
1. A place to drop random notes for future consideration into their own topic area.

**Comment by barrycumbie** [comment]
@KadotaTerry  says make sure you @ someone. 

comments on this gist? on an issue? ... wiki? wheverever?

**Comment by KadotaTerry** [comment]
Combine this issue with issue #29 . That group is already taking care of this problem along with others just like it.
